zerver/webhooks/freshstatus/doc.md

```diff
@@ -0,0 +1,79 @@
+Receive Freshstatus notifications in Zulip!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to your Freshstatus dashboard. Click **Settings** and then
+   **Integrations**. Go to **Webhooks** and click **Manage** and
+   then click **New Webhook**.
+
+1. Under **Webhook Name** enter `Zulip` or anything else you wish.
+   Enter whatever you like under **Description**.
```

Comment:
Set **Webhook Name** to any name of your choice, such as `Zulip`. Set **Description** to any suitable description.